data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1479 @@
+
+	on offer
+		conversation
+			`As you step off your ship, you are greeted by an out-of-breath Jane Melgrew. "Captain <last>!" she exclaims. "Glad I caught you. I have another job, and I couldn't just send you a message for this one.`
+			choice
+				`	"What's going on?"`
+				`	"Sorry, I'm really busy."`
+					decline
+			`	"We have reason to believe there is a ship loaded with alien tech somewhere in the south," she says. "As I'm sure you've heard, the worlds in this area have been organizing a rebellion. So far they've been peaceful, but if they are arming themselves with alien technology, that could be a major problem for us. Would you be able to track down the ship and scan it?"`
+			choice
+				`	"Sure, that sounds simple enough."`
+				`	"Sorry, that sounds way too dangerous."`
+					decline
+			`	"Fantastic. It's a Falcon named the <npc>, last seen near Yed Prior. If it is alien technology, we'll be the first in the Republic to see it, how exciting."`
+				accept
+
+	on accept
+		log "Volunteered to scan a ship in Free Worlds space that might be armed with alien technology."
+
+	npc "scan outfits"
+		personality staying uninterested
+		government "Free Worlds"
+		system "Yed Prior"
+		fleet
+			names "free worlds capital"
+			variant
+				"Falcon (Heavy)"
+		conversation
+			`As soon as you scan the <npc>, its captain hails you. "Excuse me, captain," he says, "mind telling me why you felt the need to probe my ship?"`
+			choice
+				`	"Your ship looked so powerful, I just had to know what kind of gear you had. Some of it looked almost alien."`
+				`	"I hope to have a ship like that someday, and I just wanted a shopping list of what I would need."`
+					goto equipment
+			`	"Alien? Well, maybe to someone flying a tub like yours," he says, chuckling. "No, I promise you this is all standard human fare."`
+				goto close
+			label equipment
+			`	He looks awfully pleased with himself. "Yes, she is quite the beauty. I would have never been able to afford anything like this without the Defense Pact's scouting program," he says. "She makes keeping pirates out of our territory as easy as a dream. If you're interested in trading that tub you've got in for something like this, the program should be hiring again soon."`
+			label close
+			`	You cut the comlink and breathe a sigh of relief that the conversation is over.`
+
+	on complete
+		"assisted republic" ++
+		dialog `Soon after you land, Jane comes to your ship and pays you <payment> in exchange for your sensor logs regarding <npc>. "No alien tech, I see," she says, sounding dissapointed. "While that would have been a great find, I will admit I'm happy I don't have to figure out how to defend against it. I've let my supervisors know how helpful you've been, hopefully we can work together again soon." `
+		payment 50000
+		log "Captain Melgrew says she has put a good word in for me with her superiors."
+
+
+
+mission "Republic Shipping 1"
+	name "Secret cargo to <destination>."
```

Comment:
```suggestion
	name "Secret cargo to <planet>"
```

Reply:
Use <planet> instead of <destination> for mission titles, otherwise they'll almost always overflow.

![image](https://user-images.githubusercontent.com/4471575/95669454-88764280-0b46-11eb-9b50-90da26e4676b.png)